pages/pipelines/rules.md

```diff
@@ -0,0 +1,169 @@
+# Pipeline rules
```

Comment:
I wonder if we should be referring to this simply as 'Rules' rather than 'Pipeline rules'? Just thinking forward to using this for things other than pipelines.